### PR TITLE
Weekly merge

### PR DESCRIPTION
**Pull request by toniocodo**
This PR contains:
- Governance: vAPY computation fix for lockup extend
- Governance: vAPY without amount feat
- Governance: update proposal count
- OETH: add OETH => sfrxETH route
- update gas price calculation
- update public images distribution
- add safepal wallet

### CODE REVIEW

**Review by nick — APPROVED** [review]

File: libs/defi/shared/src/clients/graphql.ts [hunk 1]
```diff
@@ -12,7 +12,6 @@ export const graphqlClient =
   ) =>
   async () => {
     const res = await axiosInstance<TData>({
-      url: '/graphql',
```
Comment (apexearth) [inline libs/defi/shared/src/clients/graphql.ts:15]: ill assume this is OK

File: libs/oeth/swap/src/actions/swapCurveSfrxeth.ts [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+const config = {
+  routes: [
+    tokens.mainnet.OETH.address,
+    contracts.mainnet.OETHCurvePool.address,
+    ETH_ADDRESS_CURVE,
+    '0xa1f8a6807c402e4a15ef4eba36528a3fed24e577',
+    tokens.mainnet.frxETH.address,
+    tokens.mainnet.sfrxETH.address,
+    tokens.mainnet.sfrxETH.address,
```
Comment (apexearth) [inline libs/oeth/swap/src/actions/swapCurveSfrxeth.ts:38]: no idea how this works, but do we mean to have this one twice?

**Review by apexearth — APPROVED** [review]
couple comments - lgtm

File: libs/oeth/swap/src/actions/swapCurveSfrxeth.ts (hunk 2 above)
Comment (toniocodo) [reply libs/oeth/swap/src/actions/swapCurveSfrxeth.ts:38]: yes, this is how curve params are prepared, it means "call stake function on sfrxETH to wrap frxETH". Great times figuring that out 😓

**Pull request merged**